inference-engine/tests/functional/plugin/shared/include/behavior/core_threading_tests.hpp

```diff
@@ -192,6 +194,12 @@ class CoreThreadingTestsWithIterations : public ::testing::TestWithParam<CoreThr
         std::tie(deviceName, config) = std::get<0>(GetParam());
         numThreads =  std::get<1>(GetParam());
         numIterations =  std::get<2>(GetParam());
+
+        isGnaTest = isGnaTest || deviceName == CommonTestUtils::DEVICE_GNA;
```

Comment:
Why do you use `GNA` device  in the shared library????

Reply:
Could you elaborate what do you mean?

Reply:
We don't use hardcode like `if (deviceName == GNA)` in shared lib

Reply:
I can add a "custom" parameter like 'useSimplifiedModel' instead and pass this parameter only from GNA test target.
Do you think it would be acceptable?
Alternatively, what is your recommendation?

Reply:
We can use the model list as a parameter for test. 
For example, we can have a basic model list which is defined in the test body, and extend it using models taken from test parameter.
Or define model list as a test parameter exactly. 
These approaches will be more flexible.

Reply:
I added new parameter. Unfortunately CI fails due to related changes needed in 
[](https://gitlab-icv.inn.intel.com/inference-engine/armnn-plugin) repo.
there is `armnn-plugin/tests/functional/shared_tests_instances/behavior/core_threading_tests.cpp` file there which need to be changed.
@iefode, @ilya-lavrenov please check whether current approach is acceptable (new parameter), so I will create PR for armnn-plugin. Or suggest some other approach.

Reply:
@ilya-lavrenov please see https://gitlab-icv.inn.intel.com/inference-engine/armnn-plugin/merge_requests/135 and merge if OK to pass CI for this one.

Reply:
CI passed (finally) :)